omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+	   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	   xsi:schemaLocation="http://www.springframework.org/schema/beans
+  		    http://www.springframework.org/schema/beans/spring-beans-3.0.xsd">
+
+    <!-- UI Framework setup -->
+    <bean class="org.openmrs.ui.framework.StandardModuleUiConfiguration">
+        <property name="moduleId" value="${project.parent.artifactId}"/>
+    </bean>
+
```

Comment:
Wires in any beans required in the OMOD layer.

We wire in a basic bean to support creating OpenMRS 2.0 UI classes within this module.  If we end up not adding any 2.0 pages, this bean could be removed (and the dependency on UI Framework removed).